go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/prometheus/prometheus v1.8.2-0.20190924101040-52e0504f83ea
 	github.com/rs/cors v1.6.0
 	github.com/securego/gosec v0.0.0-20200316084457-7da9f46445fd
-	github.com/shirou/gopsutil v2.20.4+incompatible
+	github.com/shirou/gopsutil v0.0.0-20200517204708-c89193f22d93 // c89193f22d9359848988f32aee972122bb2abdc2
```

Comment:
`gopsutil` needed to be updated to include this relatively recent PR that improves the performance of reading process information on Windows: https://github.com/shirou/gopsutil/pull/862